src/helpers/fileSizeHelper.ts

```diff
@@ -9,3 +9,14 @@ export const formatSize = (size: number) => {
     return `${(size / 1000000000).toFixed(1)} GB`;
   }
 };
+
+// thanks for : https://gist.github.com/zentala/1e6f72438796d74531803cc3833c039c
+export const formatSizeForAugmentedReality = (bytes: number, decimals = 1) => {
+  if (bytes == 0) return '0 Bytes';
+
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
`'TB', 'PB', 'EB', 'ZB', 'YB'` 🤣 this wpuld be huuuuge files for augmented reality ;) up to `'GB'` is enough for us i guess?

Reply:
Yeah, it's a lot.

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/670/commits/608cef5e83b81948dcdf109744e2edc1f44fd55c